### PR TITLE
Issue288

### PR DESCRIPTION
**Pull request by humphd**

### CODE REVIEW

File: src/brackets.js [hunk 1]
```diff
@@ -151,69 +151,6 @@ define(function (require, exports, module) {
     // read URL params
     params.parse();
     
-
-    /**
-     * Setup test object
-     */
-    function _initTest() {
```
Comment (sedge) [inline src/brackets.js]: This set up the in-brackets tests ya? Which we don't care about?

File: src/LiveDevelopment/LiveDevMultiBrowser.js [hunk 2]
```diff
@@ -828,12 +828,6 @@ define(function (require, exports, module) {
         ProjectManager
             .on("beforeProjectClose beforeAppClose", close);
         
-        // Default transport for live connection messages - can be changed
-        setTransport(NodeSocketTransport);
```
Comment (sedge) [inline src/LiveDevelopment/LiveDevMultiBrowser.js]: Oooh! And we get rid of the `NodeSocketTransport` initializing as well!

**Comment by sedge** [comment]
r+ w/ travis fixed

File: src/brackets.js (hunk 1 above)
Comment (humphd) [inline src/brackets.js]: Correct.

**Comment by humphd** [comment]
This is passing Travis now, and works well (it drops about ~300K from the size of Brackets, pre gzipping).  I'd love to get some feedback from @busykai before I land, though.

**Comment by busykai** [comment]
@humphd, this seems about right to me. you are not loading the entire CDT-based LiveDevelopment and test dependencies. could not test the correctness, though, as i don't have a setup, but this is what i was imagining needs to be done to improve the start-up time (+ you found the test part!).

**Comment by humphd** [comment]
Thanks @busykai, this is helpful.  I'm OK with breaking tests, since they aren't really runnable in browser anyway, and we can/do run them on `master`.  I'm going to rebase and land this here.

**Pull request merged**